### ISSUE TITLE
`import ollama from 'npm:ollama@0.4.2';` from within deno throws an error

### ISSUE THREAD

**Issue by nullstyle** [ollama/ollama-js#26]
Given the script:
```typescript
import ollama from 'npm:ollama@0.4.2';

const response = await ollama.chat({
  model: 'llama2',
  messages: [{ role: 'user', content: 'Why is the sky blue?' }],
})
console.log(response.message.content)
```

When running:
`deno run --allow-net --allow-read test.ts`

Then I get:
```
error: Could not resolve './interfaces' from 'file:///Users/nullstyle/Library/Caches/deno/npm/registry.npmjs.org/ollama/0.4.2/dist/index.js'.

Caused by:
    [ERR_MODULE_NOT_FOUND] Cannot find module "file:///Users/nullstyle/Library/Caches/deno/npm/registry.npmjs.org/ollama/0.4.2/dist/interfaces" imported from "file:///Users/nullstyle/Library/Caches/deno/npm/registry.npmjs.org/ollama/0.4.2/dist/index.js"
```

**Comment by Leask** [comment]
same here.

**Comment by BruceMacD** [comment]
Thanks for the report, this should be fixed now in `v0.4.3`

**Comment by nullstyle** [comment]
Beautiful!  Thank you so much @BruceMacD
